### PR TITLE
govendor: don't sync dependencies if `.heroku.sync` == false

### LINKED ISSUE #149: govendor: can't easily manually update vendors
Hello there,

Locally to my `/vendor/` dir I am trying to do little changes to a library that _might_ have issues.

But the buildpack will always do 

```
-----> Fetching any unsaved dependencies (govendor sync)
```

which will always replace those little changes I made.

This can be avoided by committing those changes somewhere, but it'd be a pain.

I also believe code freedom inside`/vendor` is really important.

Is it possible to avoid this from happening ?
May be by adding  a `heroku.sync` bool in **vendors.json** ?

### PR DESCRIPTION
**Pull request by azr**
this fixes #149

### CODE REVIEW

**Comment by azr** [comment]
probably, other dep managers need this ?

File: bin/compile [hunk 1]
```diff
@@ -515,8 +515,11 @@ case ${TOOL} in
         unset GIT_DIR # unset git dir or it will mess with goinstall
         cd $p
 
-        step "Fetching any unsaved dependencies (govendor sync)"
-        govendor sync
+        if [test `<${vendorJSON} jq -r '.heroku.sync'` != false]
```
Comment (freeformz) [inline bin/compile]: Drop the `[` and `]`

**Comment by freeformz** [comment]
Glide would need a config var. GB/godep doesn't have this type of step.

File: bin/compile (hunk 1 above)
Comment (azr) [reply bin/compile]: done ! :)

**Comment by azr** [comment]
okay

**Comment by azr** [comment]
Hey, hem, I dont really understand if I broke the tests ?

**Review by freeformz — APPROVED** [review]

**Comment by freeformz** [comment]
:+1:

**Comment by freeformz** [comment]
Thanks for the contribution. I'll likely release this tomorrow.

**Comment by azr** [comment]
Nice !! :)

**Pull request merged**